### PR TITLE
Added cog for Halloween facts

### PR DESCRIPTION
**Pull request by meooow25**
For issue #32 

The new cog
* Posts a random Halloween fact to a channel at regular intervals, default interval set to 6 hours.
* Displays last posted fact on command `hallofact`.

demo

### CODE REVIEW

File: bot/cogs/halloween_facts.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+        self.bot.loop.create_task(self._fact_publisher_task())
+
+    async def _fact_publisher_task(self):
+        """
+        A background task that runs forever, sending Halloween facts at random to the Discord channel with id equal to
+        HACKTOBERFEST_CHANNEL_ID every INTERVAL seconds.
+        """
+        await self.bot.wait_until_ready()
+        self.channel = self.bot.get_channel(HACKTOBERFEST_CHANNEL_ID)
```
[the remark this answers is not in the document]
Comment (meooow25) [reply bot/cogs/halloween_facts.py]: Thanks, done.

File: bot/cogs/halloween_facts.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    "\N{DERELICT HOUSE BUILDING}",
+    "\N{EXTRATERRESTRIAL ALIEN}",
+    "\N{GHOST}",
+    "\N{JACK-O-LANTERN}",
+    "\N{SKULL}",
+    "\N{SKULL AND CROSSBONES}",
+    "\N{SPIDER WEB}",
+]
+PUMPKIN_ORANGE = discord.Color(0xFF7518)
+HACKTOBERFEST_CHANNEL_ID = 101010  # Replace with actual channel ID.
```
Comment (sco1) [inline bot/cogs/halloween_facts.py]: `498804484324196362` is Hacktoberbot's channel

**Comment by markylon** [comment]
Why make a new variable for JSON file name? You're only using it once and it's not like it changes often.

File: bot/cogs/halloween_facts.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+PUMPKIN_ORANGE = discord.Color(0xFF7518)
+HACKTOBERBOT_CHANNEL_ID = 498804484324196362
+INTERVAL = timedelta(hours=6).total_seconds()
+
+
+class HalloweenFacts:
+
+    def __init__(self, bot):
+        self.bot = bot
+        with open(Path("resources", "halloween_facts.json"), "r") as file:
```
Comment (jos-b) [inline bot/cogs/halloween_facts.py]: This currently errors as we changed some of the structure a couple days ago.

This line should instead use:
```py
Path("./bot/resources", "halloween_facts.json")
```

**Comment by meooow25** [comment]
Updated resources location as requested.

**Comment by jos-b** [comment]
Okay! This looks good!

**Pull request merged**